Description: set default FDT filenames to load from boot disk
Bug: https://github.com/sifive/meta-sifive/issues/16

Index: u-boot-2021.01+dfsg/configs/sifive_fu540_defconfig
===================================================================
--- u-boot-2021.01+dfsg.orig/configs/sifive_fu540_defconfig
+++ u-boot-2021.01+dfsg/configs/sifive_fu540_defconfig
@@ -1,3 +1,4 @@
+CONFIG_DEFAULT_FDT_FILE="sifive/hifive-unleashed-a00.dtb"
 CONFIG_RISCV=y
 CONFIG_SPL_GPIO_SUPPORT=y
 CONFIG_SYS_MALLOC_F_LEN=0x3000
Index: u-boot-2021.01+dfsg/configs/sifive_hifive_unmatched_fu740_defconfig
===================================================================
--- u-boot-2021.01+dfsg.orig/configs/sifive_hifive_unmatched_fu740_defconfig
+++ u-boot-2021.01+dfsg/configs/sifive_hifive_unmatched_fu740_defconfig
@@ -1,3 +1,4 @@
+CONFIG_DEFAULT_FDT_FILE="sifive/hifive-unmatched-a00.dtb"
 CONFIG_RISCV=y
 CONFIG_SPL_GPIO_SUPPORT=y
 CONFIG_SYS_MALLOC_F_LEN=0x3000
Index: u-boot-2021.01+dfsg/include/configs/sifive-fu540.h
===================================================================
--- u-boot-2021.01+dfsg.orig/include/configs/sifive-fu540.h
+++ u-boot-2021.01+dfsg/include/configs/sifive-fu540.h
@@ -72,6 +72,7 @@
 	"type_guid_gpt_loader2=" TYPE_GUID_LOADER2 "\0" \
 	"type_guid_gpt_system=" TYPE_GUID_SYSTEM "\0" \
 	"partitions=" PARTS_DEFAULT "\0" \
+	"fdtfile=" CONFIG_DEFAULT_FDT_FILE "\0" \
 	BOOTENV \
 	BOOTENV_SF
 
Index: u-boot-2021.01+dfsg/include/configs/sifive-hifive-unmatched-fu740.h
===================================================================
--- u-boot-2021.01+dfsg.orig/include/configs/sifive-hifive-unmatched-fu740.h
+++ u-boot-2021.01+dfsg/include/configs/sifive-hifive-unmatched-fu740.h
@@ -84,6 +84,7 @@
 	"type_guid_gpt_loader2=" TYPE_GUID_LOADER2 "\0" \
 	"type_guid_gpt_system=" TYPE_GUID_SYSTEM "\0" \
 	"partitions=" PARTS_DEFAULT "\0" \
+	"fdtfile=" CONFIG_DEFAULT_FDT_FILE "\0" \
 	BOOTENV
 
 #define CONFIG_PREBOOT \
